zebra-network/src/address_book_updater.rs

```diff
@@ -13,6 +13,9 @@ use crate::{
     address_book::AddressMetrics, meta_addr::MetaAddrChange, AddressBook, BoxError, Config,
 };
 
+/// The minimum size of the address book updater channel.
+pub const MIN_CHANNEL_SIZE: usize = 10;
```

Comment:
Optional/Nitpick: Use a power of 2 as the default minimum for channel capacities where appropriate if you think that would be a good convention.

Reply:
I don't really mind either way.